examples/deployment-manager/autogen/multivm/README.md

```diff
@@ -1,84 +1,73 @@
+
+To generate a Deployment Manager template for your VM product, 
+see the Google Cloud Marketplace documentation for [Creating your deployment package](https://cloud.google.com/marketplace/docs/partners/vm/create-deployment-package).
+
 ## Prerequisites
 
-This example package generates a Deployment Manager template for a Redis 
-multiple VM setup. Your GCP project must contain a Redis VM image to use in the
-Deployment Manager template. The following command copies the Redis VM image
-from the
-[click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
-to your GCP project.
+Your Google Cloud project must contain a Redis VM image to use in the
+Deployment Manager template. Run the following command to copy the Redis VM image
+from the [click-to-deploy project](https://github.com/GoogleCloudPlatform/click-to-deploy)
+to your project:
 
 ```
 PROJECT_ID=<YOUR_PROJECT_ID>
 gcloud compute --project=$PROJECT_ID images create redis --source-image=redis-v20200726 --source-image-project=click-to-deploy-images
 ```
 
-## Generating Deployment Manager Template
+## Generate a Deployment Manager template
 
-To use this `mpdev` package, execute the following command which will checkout this
-package to a directory named `redis`.
+To retrieve this example's Autogen specification, execute the following command which checks out the
+specification to a directory named `redis`:
 
 ```
 mpdev pkg get https://github.com/GoogleCloudPlatform/marketplace-tools.git/examples/deployment-manager/autogen/multivm/ redis
 ```
 
-`mpdev cfg set` will be used to customize values in the 
-configurations.yaml template. Particularly the
+### Update the Autogen specification
+
+You use the command `mpdev cfg set` to update the configurations.yaml file with your product's
 [`projectId`](../../../../docs/autogen-reference.md#cloud.deploymentmanager.autogen.ImageSpec)
 and [`image`](../../../../docs/autogen-reference.md#cloud.deploymentmanager.autogen.ImageSpec)
-values must be set for your particular solution.
+values.
+
+Set the variables for the Google Cloud project and name of the VM image:
 
 ```
 PROJECT_ID=<PROJECT_ID>
 IMAGE=redis
 ```
 
-After setting the variables above run the following commands to set the
-variables in `configurations.yaml`
+Next, run the following commands to update the values in `configurations.yaml`:
 
 ```bash
 mpdev cfg set redis/ projectId $PROJECT_ID
 mpdev cfg set redis/ image $IMAGE
 mpdev cfg set redis/ defaultReplicas 4
-
-# Optionally set deployment package zip output path.
-# Set to GCS location (e.g. gs://<BUCKET>/<OBJECT>) if using 
-# Producer Portal (Public Preview)
-mpdev cfg set redis/ zipPath template.zip
 ```
 
-Now generate a Deployment Manager template with the following command:
+### Generate the Deployment Manager template
+
+With the updated Autogen specification, run the following command to generate a Deployment Manager template:
```

Comment:
> With the updated Autogen specification, run the following command to generate a Deployment Manager template:

To generate a Deployment Manager template, run the following command with your updated Autogen specification: